Release Notes/2017/2017-05-23-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,48 @@
+{{{
+"title": "Cloud Platform - Release Notes: May 23, 2017",
+"date": "5-23-2017",
+"author": "Bob Jackson",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### New Features (1)
+
+* __Cloud Application Manager__
+
+- Bulk Import of auto discovered instances
+Cloud Application Manager now allows users to register auto discovered instances in bulk. Full instructions on how to use this new feature are available at the Knowledge Base section on our website.
+
+- Microsoft Azure Resale available for U.K. based customers
+Azure resale to companies based in the United Kingdom is Live! From the Azure Provider, users whose main registered address with CenturyLink is in the United Kingdom, will be permitted to automatically provision new Azure accounts for consolidated billing and platform level support. The capabilities that CenturyLink will Resell will still be the same as defined here https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-azure-capabilities/
+
+
+### Enhancements (3)
+
+* __Load Balancer as a Service__
+
+Load Balancer as a Service has added URI as a supported load balancing method. This new capability is now available in addition to Least Connection, Round Robin, and Source IP support.
+
+* __Managed Applications__
+
+Application Product Engineering has tested and certified Apache Tomcat
+7.0.77, 8.0.43 and 8.5.14 on RHEL  6 and 7.
+
+The Software Policy to use for RHEL 6 and 7 have been combined into a single policy.  The script which installs the package is now able to detect which major version of RHEL is targeted for the install, and utilizes the appropriate RHN channel for the installation.
+
+Installation Instructions: https://confluence.savvis.net/pages/viewpage.action?pageId=114091033

```

Comment:
this is not a valid path to a public location for customers.  please adjust to an actual KB article to which this applies.  links to private CTL assets is not for public kbs